README.md

```diff
@@ -161,8 +171,11 @@ make u
 ## Environment Variables
 
 | **Environment Variable**|**Description**|**Usage**|**Error Handling**|
-|---|---|---|---|
-| `CSH_FOCA_CONFIG_PATH` | Specifies the path to the configuration file for the FOCA app. | The FOCA app uses this environment variable to locate the configuration file. If not set, it defaults to dev. | If the configuration file is not found at the specified or default path, a `FileNotFoundError` will be raised. |
+|---|---|---|---| | `CSH_FOCA_CONFIG_PATH` | Specifies the path to the
+configuration file for the FOCA app. | The FOCA app uses this environment
+variable to locate the configuration file. If not set, it defaults to dev. | If
+the configuration file is not found at the specified or default path, a
+`FileNotFoundError` will be raised. |
```

Comment:
The table doesn't render. Make sure to put `| --- | --- | --- | --- |` on its own line, then put each table row (I suppose there is only one at this point) in their own very long lines. I know this breaks markdown linting, but you will need to set an exemption here, because you can't wrap table lines or else the table won't be rendered.